src/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -2,12 +2,12 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsNetStandard)' == 'true' and '$(TargetsWindows)' != 'true' and '$(TargetsLinux)' != 'true' and '$(TargetsOSX)' != 'true'">SR.PlatformNotSupported_IOPorts</GeneratePlatformNotSupportedAssemblyMessage>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsNetStandard)' == 'true' and '$(TargetsWindows)' != 'true' and '$(TargetsLinux)' != 'true' and '$(TargetsOSX)' != 'true' and '$(TargetsFreeBSD)' != 'true'">SR.PlatformNotSupported_IOPorts</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
What configuration does this still apply to?

Reply:
I think the UnknownUnix  was originally there as stub for new platforms. Web assembly may fall into this bucket now.  cc @safern

Reply:
Not necessary for this PR, but this condition is getting a bit unwieldy... send like it could be better to start listing what should have this done rather than what shouldn't, or find some other way to simplify it.
cc: @krwq

Reply:
Currently this project (with this change) has the following configurations: 

https://github.com/dotnet/corefx/blob/6e190fe8f2b031f1434f18fb3125bb2a3963fc74/src/System.IO.Ports/src/Configurations.props#L3-L14

If we look at the os groups definitions:
https://github.com/dotnet/corefx/blob/6e190fe8f2b031f1434f18fb3125bb2a3963fc74/eng/configurations/osgroups.props#L13-L43

So basically this configuration still applies for `AnyOS` which is the `netstandard2.0` OS agnostic build configuration. We could update this condition to be:

`'$(TargetsNetStandard)' == 'true' and '$(OSGroup)' == 'AnyOS'`